esp/esp/program/modules/handlers/onsiteprintschedules.py

```diff
@@ -56,7 +56,7 @@ def module_properties(cls):
             }
 
     @main_call
-#    @needs_onsite
+    @needs_onsite
```

Comment:
It didn't work, but not because of @needs_onsite but because latex.py doesn't support ps output, just pdf, log, tex, svg, and png. I pushed a fix.